### DISCUSSION TITLE
Change appearance of citations in search results

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4971] [General]
Thanks Teresa! We'll go ahead and use these citation types. I have one more comment about it though. We are not crazy about how the citation type appears somewhat prominently below the catalog number on the search results page. I was wondering what the possibilty would be of having it read 1 Citation much like it reads 2 Events ? 

![Screen Shot 2020-01-10 at 12 46 19 PM](https://user-images.githubusercontent.com/40579515/72188758-958fb680-33a7-11ea-919f-69312d47b97d.png)

_Originally posted by @krgomez in https://github.com/ArctosDB/arctos/issues/1861#issuecomment-573218844_

**Comment by marecaguthrie** [comment]
To expand on Karinna's concern - the current display implies that either
the artwork is the inspiration for a different illustration or that the
artwork forms the foundation for an illustration, The term"Illustration"
has a very specific connotation as a category of art that I'm afraid will
make it confusing for users. "1 citation" or 2 citations" seems like a more
accurate description.

On Fri, Jan 10, 2020 at 2:10 PM Teresa Mayfield-Meyer <
notifications@github.com> wrote:

> Thanks Teresa! We'll go ahead and use these citation types. I have one
> more comment about it though. We are not crazy about how the citation type
> appears somewhat prominently below the catalog number on the search results
> page. I was wondering what the possibilty would be of having it read 1
> Citation much like it reads 2 Events ?
>
> [image: Screen Shot 2020-01-10 at 12 46 19 PM]
> <https://user-images.githubusercontent.com/40579515/72188758-958fb680-33a7-11ea-919f-69312d47b97d.png>
>
> *Originally posted by @krgomez <https://github.com/krgomez> in #1861
> (comment)
> <https://github.com/ArctosDB/arctos/issues/1861#issuecomment-573218844>*
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ArctosDB/arctos/issues/2438?email_source=notifications&email_token=AJKSRRYALTQIDA2Q6RXNZATQ5D56XA5CNFSM4KFOOHQKYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4IFO5BFQ>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJKSRR3KYYD32AVGFCJPXBTQ5D56XANCNFSM4KFOOHQA>
> .
>


-- 
Mareca Guthrie
Curator of Fine Arts & Associate Professor of Art
University of Alaska Museum of the North
1962 Yukon Drive
P.O. Box 756960
Fairbanks, AK 99775-6960
mrguthrie@alaska.edu

University of Alaska Museum of the North: www.uaf.edu/museum
UAF Art Department: https://www.uaf.edu/art/ <https://www.uaf.edu/art/>
Colors of Nature: http://www.colorsofnature.org/

**Comment by dustymc** [comment]
Merging back to https://github.com/ArctosDB/arctos/issues/1861

**Comment by krgomez** [comment]
Is it critical for science collections to see what type of citation is referenced in a catalog record in the search results?

**Comment by ebraker** [comment]
It is very helpful to see if a citation is a type specimen on the biological side of things, though this info is also viewable under the "citations" column in the form. However, outside users may not customize the results form and may miss this info. Perhaps we could do a generic citation count as suggested above, but if citation_type = any "type", also display the specific value below GUID.
